src/index.js

```diff
@@ -150,7 +152,7 @@ export default function supernova(env) {
               });
           }
         }
-      }, [element, dataTree]);
+      }, [element, dataTree, constraints]);
```

Comment:
since we listen to `constraints` here, we probably don't need to have it in other `useEffects`, since they listen to `objectData`

Reply:
Yes good one!

Reply:
Actually found a way to keep the zoom/panning between edit/analysis mode. Changes the filter function though, so something you might want to have a look at